model/host/volume.go

```diff
@@ -157,6 +170,16 @@ func FindTotalVolumeSizeByUser(user string) (int, error) {
 	return out[0].TotalVolumeSize, nil
 }
 
+func FindVolumesWithNoExpirationToExtend() ([]Volume, error) {
+	query := bson.M{
+		VolumeNoExpirationKey: true,
+		VolumeHostKey:         "",
```

Comment:
Why should the volume have no host?

Reply:
the host key is empty when a volume isn't attached to a host 
(never been attached, detached, or the host is terminated)